cmd/common.go

```diff
@@ -58,6 +68,110 @@ func addKeys(config config.Configuration, idService identity.Service) error {
 	return nil
 }
 
+func getKeyPairsFromConfig(config config.Configuration) (map[int]did.Key, error) {
```

Comment:
I believe this function should be more in the identity package as it will be reused from the API as well?